lib/plugins/aws/invokeLocal/index.js

```diff
@@ -157,7 +158,8 @@ class AwsInvokeLocal {
     });
 
     if (process.env.VIRTUAL_ENV) {
-      process.env.PATH = `${process.env.VIRTUAL_ENV}/bin:${process.env.PATH}`;
+      const runtimeDir = os.platform() === 'win32' ? 'Scripts' : 'bin';
+      process.env.PATH = `${path.join(process.env.VIRTUAL_ENV, runtimeDir)}:${process.env.PATH}`;
```

Comment:
Also, the env PATH on Windows uses `;` as a separator, not `:`... I think `path.delimiter` should do it, https://nodejs.org/api/path.html#path_path_delimiter.

Reply:
Thanks!